src/layers/core/hexagon-layer/hexagon-aggregator.js

```diff
@@ -45,10 +45,10 @@ export function pointToHexbin({data, radius, getPosition}, viewport) {
 
   const hexagonBins = newHexbin(screenPoints);
 
-  return hexagonBins.map(hex => ({
+  return {hexagons: hexagonBins.map(hex => ({
```

Comment:
Nit: add a newline to make the object structure more visible:

    return {
        hexagons: ....
    };